### PR TITLE
Update azure blob storage binding

### LINKED ISSUE #808: Update Azure Blob Binding
Update the specs for Azure Blob Binding with the following information:

1. New GET operation for fetching a blob
2. New response structure from CREATE operation with the blob url

### PR DESCRIPTION
**Pull request by yaron2**
Closes #808

### CODE REVIEW

File: reference/specs/bindings/blobstorage.md [hunk 1]
```diff
@@ -28,9 +28,64 @@ spec:
...
+  "data": {
+    "field1": "value1"
+  }
+}
+```
+
+Example:
+
+```bash
+curl -XPOST http://localhost:<dapr-port>/v1.0/bindings/<binding-name> -d '{ "operation": "create", "data": { "field1": "value1" }}'
```
Comment (mchmarny) [inline reference/specs/bindings/blobstorage.md:51]: nit, data first for readability. Also `POST` is implicit with `-d`

```shell
curl -d '{ "operation": "create", "data": { "field1": "value1" }}' \
       http://localhost:<dapr-port>/v1.0/bindings/<binding-name>
```

File: reference/specs/bindings/blobstorage.md [hunk 2]
```diff
@@ -28,9 +28,64 @@ spec:
...
+  "metadata": {
+    "blobName": "myblob"
+  }
+}
+```
+
+Example:
+
+```bash
+curl -XPOST http://localhost:<dapr-port>/v1.0/bindings/<binding-name> -d '{ "operation": "get", "metadata": { "blobName": "myblob" }}'
```
Comment (mchmarny) [inline reference/specs/bindings/blobstorage.md:81]: same as above

Comment (yaron2) [reply reference/specs/bindings/blobstorage.md]: Changed.

File: reference/specs/bindings/blobstorage.md (hunk 1 above)
Comment (yaron2) [reply reference/specs/bindings/blobstorage.md]: Changed.

**Review by mchmarny — APPROVED** [review]
/lgtm

**Pull request merged**